trimesh/exchange/gltf.py

```diff
@@ -1463,7 +1462,6 @@ def _read_buffers(header,
                     # each primitive gets it's own Trimesh object
                     if len(m["primitives"]) > 1:
                         kwargs['metadata']['from_gltf_primitive'] = True
-                        name = unique_name(name, meshes)
```

Comment:
`name` is computed earlier in the loop and it doesn't look like there are any changes to `name` or `meshes` in between line 1394 and here